modules/duplicating-a-scheduled-pipeline-run.adoc

```diff
@@ -17,34 +16,30 @@ ifdef::upstream[]
-. In the *Description* field, enter a description for the run that you want to duplicate.
-. From the *Pipeline* list, select the pipeline containing the run that you want to duplicate.
-. From the *Pipeline version* list, select the pipeline version containing the run that you want to duplicate.
-. To configure the run type for the run that you are duplicating, in the *Run type* section, perform one of the following sets of actions:
-* Select *Run once immediately after create* to specify the run that you are duplicating executes once, and immediately after its creation. If you selected this option, skip to step 10.
-* Select *Schedule recurring run* to schedule the run that you are duplicating to recur.
-. If you selected *Schedule recurring run* in the previous step, to configure the trigger type for the run, perform one of the following actions:
-* Select *Periodic* and select the execution frequency from the *Run every* list.
+. On the *Runs* page, select the *project* that has the pipeline run that you want to duplicate.
+. Click the *Schedules* tab.
+. Click the action menu (*&#8942;*) beside the run that you want to duplicate and click *Duplicate*.
+. On the *Duplicate schedule* page, configure the duplicate run:
+.. In the *Name* field, enter a name for the duplicate run.
+.. In the *Description* field, enter a description for the duplicate run.
+.. From the *Trigger type* list, select one of the following options:
+* Select *Periodic* to specify an execution frequency. In the *Run every* field, enter a numerical value and select an execution frequency from the list.
 * Select *Cron* to specify the execution schedule in `cron` format. This creates a cron job to execute the run. Click the *Copy* button (image:images/osd-copy.png[]) to copy the cron job schedule to the clipboard. The field furthest to the left represents seconds. For more information about scheduling tasks using the supported `cron` format, see link:https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format[Cron Expression Format].
-. If you selected *Schedule recurring run* in step 7, configure the duration for the run that you are duplicating.
-.. Select the *Start date* check box to specify a start date for the run. Select the start date using the calendar tool and the start time from the list of times.
-.. Select the *End date* check box to specify an end date for the run. Select the end date using the calendar tool and the end time from the list of times.
-. In the *Parameters* section, configure the input parameters for the run that you are duplicating by selecting the appropriate parameters from the list.
-. Click *Create*.
+.. For *Maximum concurrent runs*, specify the number of runs that can execute concurrently, from a range of one to ten.  
+.. For *Start date*, specify a start date for the duplicate run. Select a start date using the *Calendar* and the start time from the list of times.
```

Comment:
Select a start date **by** using the *Calendar*.  **Select** the start time from the list of times.